### PR TITLE
bullet

### PR DESCRIPTION
**Pull request by ring630**
Enhanced crreat_padstack to support bullet shape antipad

### CODE REVIEW

**Comment by maxcapodi78** [comment]
@ring630 the PR looks good to me. Two comments:
1- it would be better to add parameters at the end of the list of existing parameters to maintain the back compatibility with existing customer code.
2- maybe should be helpful to add a unit test for bullet

**Comment by MaxJPRey** [comment]
@ring630 I would just suggest to rename the parameters in order to follow the snake_case convention.

**Comment by ring630** [comment]
> @ring630 the PR looks good to me. Two comments: 1- it would be better to add parameters at the end of the list of existing parameters to maintain the back compatibility with existing customer code. 2- maybe should be helpful to add a unit test for bullet

Done.

**Comment by ring630** [comment]
> @ring630 I would just suggest to rename the new parameters (the one added in the signature of `create_padstack()`) in order to follow the snake_case convention.

Done

File: pyaedt/edb_core/padstack.py [hunk 1]
```diff
@@ -281,7 +283,20 @@ def create_padstack(
...
+        y_size : str, optional
+            Only applicable to bullet shape. The default is ``"600um"``.
+        corner_radius :
+            Only applicable to bullet shape. The default is ``"300um"``.
+        offset_x : str, optional
+            X offset of antipad. The default is ``"0.0"``.
+        offset_y : str, optional
+            Y offset of antipad. The default is ``"0.0"``.
+        rotation : str, optional
+            rotation of antipad. The default is ``"0.0"``.
```
Comment (MaxJPRey) [inline pyaedt/edb_core/padstack.py:299]:
```suggestion
            Rotation of antipad. The default is ``"0.0"``.
```

**Review by svandenb-dev — APPROVED** [review]
good for me

**Pull request merged**